packages/audiofileplayer/android/src/main/java/com/google/flutter/plugins/audiofileplayer/AudiofileplayerService.java

```diff
@@ -211,14 +209,16 @@ private void setSmallIconId() {
               .getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
       Bundle bundle = ai.metaData;
 
-      if (bundle != null) {
-        iconUri = bundle.getString("ic_audiofileplayer", iconUri);
+      if (bundle != null && bundle.containsKey("ic_audiofileplayer")) {
+        iconUri = bundle.getString("ic_audiofileplayer");
       }
     } catch (Throwable t) {
-      Log.d(TAG, "Error reading meta-data tag 'ic_audiofileplayer'. Using the App Icon instead.");
+      Log.d(
+          TAG,
+          "Something went wrong and/or there is no 'ic_audiofileplayer' in the metadata to load. Using the App Icon instead.");
```

Comment:
Remove "Something went wrong and/or". Just "There is no..."

Reply:
done